app/controllers/application_controller.rb

```diff
@@ -1,10 +1,142 @@
 require './config/environment'
+require 'pry'
 
 class ApplicationController < Sinatra::Base
+  register Sinatra::ActiveRecordExtension
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
   end
 
+  get '/' do
+  	erb :index
+  end
+
+  get '/tweets' do
+  	if logged_in?
+  		erb :'/tweets/tweets'	
+ 	else
+	  	redirect '/login'
+  	end
+  end
+
+  get '/tweets/new' do
+  	if logged_in?
+	  	erb :'/tweets/create_tweet'
+	 else
+	  	redirect '/login'
+	 end
+  end
+
+  post '/tweets' do
+  	if params[:content] != ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.